### PR TITLE
Import New Venues #580

### PR DESCRIPTION
**Pull request by nelsonic**
This PR has:
+ [x] _comprehensive_ step-by-step instructions for importing venue data #580 
+ [x] additional rows in `new_venues.exs` for the three new chains:
  + [x] Bermondsey Pub Company
  + [x] Brewdog
  + [x] Craft-Union
image

+ [x] @RobStallion updated the `new_venues.exs` script to ensure Venue Manager (users) are inserted. 🎉

### CODE REVIEW

**Comment by nelsonic** [comment]
Continue with #580  (_tomorrow_):
+ https://drive.google.com/drive/folders/1_WydEWEusLirceD1E1sT9vRrg8J_Jkea
+ [docs.google.com/spreadsheets/d/1Kl4BdM1NfkZsWrkyWgZhwa8Cv49HzvxuXEm_nZWWK-M](https://docs.google.com/spreadsheets/d/1Kl4BdM1NfkZsWrkyWgZhwa8Cv49HzvxuXEm_nZWWK-M)
+ https://dashboard.heroku.com/apps/club-soda-guide

**Comment by nelsonic** [comment]
No errors importing the data, but got a couple of warnings:
image

Going to remove the aliases.

File: priv/repo/new_venues.exs [hunk 1]
```diff
@@ -198,10 +167,94 @@ defmodule CsGuide.NewVenues do
...
+    if Map.has_key?(venue, :email) && venue.email != "" do
+      Map.put(map, :users, %{"0" => %{"email" => venue.email, "role" => "venue_admin"}})
+    else
+      map
+    end
+  end
+
+  # some venues don't have a phone_number so don't attempt to String.replace!
+  defp sanitise_phone_number(map, venue) do
+    # IO.inspect(map, label: "map")
```
Comment (RobStallion) [inline priv/repo/new_venues.exs]: Are these inspects needed?

File: priv/repo/new_venues.exs [hunk 2]
```diff
@@ -198,10 +167,94 @@ defmodule CsGuide.NewVenues do
...
+      Map.update!(map, :phone_number, fn s ->
+        to_string(s)
+        |> String.replace(" ", "")
+        |> String.trim() end)
+    else
+      map
+    end
+  end
+
+  defp sanitise_url(map, venue, key) do
```
Comment (RobStallion) [inline priv/repo/new_venues.exs]: Nice function 👍

File: priv/repo/new_venues.exs [hunk 3]
```diff
@@ -198,10 +167,94 @@ defmodule CsGuide.NewVenues do
...
+        venue
+    end
+  end
 end
 
 System.get_env("IMPORT_FILES_DIR")
 |> File.ls!()
 |> Enum.each(fn f ->
-  CsGuide.NewVenues.import_venues(File.read!("#{System.get_env("IMPORT_FILES_DIR")}/#{f}"), f)
+  if f =~ ".csv" do
```
Comment (RobStallion) [inline priv/repo/new_venues.exs]: 👍

File: priv/repo/new_venues.exs [hunk 4]
```diff
@@ -1,9 +1,9 @@
 defmodule CsGuide.NewVenues do
   alias NimbleCSV.RFC4180, as: CSV
   alias CsGuide.Resources.{Brand, Drink, Venue}
-  alias CsGuide.Categories.{DrinkType, DrinkStyle, VenueType}
+  alias CsGuide.Categories.{VenueType} # DrinkType, DrinkStyle, # git.io/fjlN5
```
Comment (RobStallion) [inline priv/repo/new_venues.exs]: 👍

File: priv/repo/new_venues.exs (hunk 1 above)
Comment (nelsonic) [reply priv/repo/new_venues.exs]: can be removed. I commented out in case they were useful for the next import.

**Pull request merged**